### PR TITLE
[doc] Refine the deployment documentation

### PR DESCRIPTION
**Pull request by Zephyrcf**
## Purpose of the pull request

Refine the deployment documentation.

## Brief change log

Refine the deployment documentation.

## Verify this pull request

Verified by manual tests.

### CODE REVIEW

File: docs/docs/en/guide/installation/pseudo-cluster.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ each machine. You could find this file in the path `bin/env/install_env.sh` and
 # ---------------------------------------------------------
 # Due to the master, worker, and API server being deployed on a single node, the IP of the server is the machine IP or localhost
 ips="localhost"
+# Modify Configurationexport sshPort="22"
```
Comment (zhongjiajie) [inline docs/docs/en/guide/installation/pseudo-cluster.md]: Not this place, I mean we should maybe add some sentences in L79 or L144, to tell users they can change env variable by `export <ENV_NAME>=<VALUE>`

Comment (zhongjiajie) [reply docs/docs/en/guide/installation/pseudo-cluster.md]: Others LGTM

File: docs/docs/zh/guide/installation/pseudo-cluster.md [hunk 2]
```diff
@@ -81,6 +81,8 @@ chmod 600 ~/.ssh/authorized_keys
 # ---------------------------------------------------------
 # Due to the master, worker, and API server being deployed on a single node, the IP of the server is the machine IP or localhost
 ips="localhost"
+# Modify Configurationexport sshPort="22"
```
Comment (EricGao888) [inline docs/docs/zh/guide/installation/pseudo-cluster.md]: Do you mean `# Modify Configuration export sshPort="22"`?

File: docs/docs/en/guide/installation/pseudo-cluster.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ Go to the ZooKeeper installation directory, copy configure file `zoo_sample.cfg`
 ## Modify Configuration
 
 After completing the preparation of the basic environment, you need to modify the configuration file according to the
-environment you used. The configuration files are both in directory `bin/env` and named `install_env.sh` and `dolphinscheduler_env.sh`.
+environment you used.Env configuration can be changed in the following ways, export <ENV_NAME>=<VALUE>, The configuration files are both in directory `bin/env` and named `install_env.sh` and `dolphinscheduler_env.sh`.
```
Comment (EricGao888) [inline docs/docs/en/guide/installation/pseudo-cluster.md]:
```suggestion
environment you used. Change the environment configurations via `export <ENV_NAME>=<VALUE>`. The configuration files are located in directory `bin/env` as `install_env.sh` and `dolphinscheduler_env.sh`.
```

File: docs/docs/en/guide/installation/pseudo-cluster.md [hunk 4]
```diff
@@ -83,6 +83,8 @@ each machine. You could find this file in the path `bin/env/install_env.sh` and
 # ---------------------------------------------------------
 # Due to the master, worker, and API server being deployed on a single node, the IP of the server is the machine IP or localhost
 ips="localhost"
+# Modify Configurationexport export sshPort="22"
```
Comment (EricGao888) [inline docs/docs/en/guide/installation/pseudo-cluster.md]: We could remove this comment as we have already had it above.

Comment (zhongjiajie) [reply docs/docs/en/guide/installation/pseudo-cluster.md]: Agree with that

File: docs/docs/en/guide/installation/pseudo-cluster.md [hunk 5]
```diff
@@ -83,6 +83,7 @@ each machine. You could find this file in the path `bin/env/install_env.sh` and
 # ---------------------------------------------------------
 # Due to the master, worker, and API server being deployed on a single node, the IP of the server is the machine IP or localhost
 ips="localhost"
+sshPort=${sshPort：-"22"}
```
Comment (zhongjiajie) [inline docs/docs/en/guide/installation/pseudo-cluster.md]:
```suggestion
sshPort="22"
```

File: docs/docs/zh/guide/installation/pseudo-cluster.md [hunk 6]
```diff
@@ -73,14 +73,15 @@ chmod 600 ~/.ssh/authorized_keys
...
-文件 `install_env.sh` 描述了哪些机器将被安装 DolphinScheduler 以及每台机器对应安装哪些服务。您可以在路径 `bin/env/install_env.sh` 中找到此文件，配置详情如下。
+文件 `install_env.sh` 描述了哪些机器将被安装 DolphinScheduler 以及每台机器对应安装哪些服务。您可以在路径 `bin/env/install_env.sh` 中找到此文件，可通过以下方式更改env变量,export <ENV_NAME>=<VALUE>，配置详情如下。
 
 ```shell
 # ---------------------------------------------------------
 # INSTALL MACHINE
 # ---------------------------------------------------------
 # Due to the master, worker, and API server being deployed on a single node, the IP of the server is the machine IP or localhost
 ips="localhost"
+sshPort=${sshPort：-"22"}
```
Comment (zhongjiajie) [inline docs/docs/zh/guide/installation/pseudo-cluster.md]:
```suggestion
sshPort="22"
```

**Review by zhongjiajie — APPROVED** [review]
LGTM, waiting for CI

**Review by EricGao888 — APPROVED** [review]

**Pull request merged**